in_toto/gpg/util.py

```diff
@@ -118,11 +117,11 @@ def parse_packet_header(data, expected_type):
     packet_length = data[1]
     ptr = 2
 
-  if packet_type != expected_type: # pragma: no cover
+  if expected_type != None and packet_type != expected_type: # pragma: no cover
```

Comment:
You should probably also document the supported values for `expected_type.`

Reply:
True, @SantiagoTorres, you mentioned that you considered removing this argument. Should we do that? Otherwise, I can add the documentation.

Reply:
I think that for now is worth documenting. The value of expected type is one of the enums [here](https://github.com/in-toto/in-toto/blob/develop/in_toto/gpg/constants.py#L27)

Reply:
Added values to docstring in 93b78d7